bitnami/metrics-server/Chart.yaml

```diff
@@ -23,4 +23,4 @@ name: metrics-server
 sources:
   - https://github.com/bitnami/bitnami-docker-metrics-server
   - https://github.com/kubernetes-incubator/metrics-server
-version: 5.8.15
+version: 5.8.16
```

Comment:
Please bump the minor version

Reply:
Updated by https://github.com/bitnami/charts/pull/6917/commits/464e25525586741f3d3e6fb2cc9401a2fa7a27c9